src/shared/schemas.ts

```diff
@@ -19,6 +19,7 @@ import { normalizeSeparator } from './utilities/pathUtils'
 import { Any, ArrayConstructor } from './utilities/typeConstructors'
 
 const GOFORMATION_MANIFEST_URL = 'https://api.github.com/repos/awslabs/goformation/releases/latest'
+const SCHEMA_PREFIX = 'aws://aws-toolkit-vscode/'
```

Comment:
should we reuse AWS_SCHEME? (and maybe lift it into constants.ts or something)? https://github.com/aws/aws-toolkit-vscode/blob/acbd172d8e0de799e880ba83aff1f6a5562f11af/src/shared/extensions/yaml.ts#L24

```suggestion
const SCHEMA_PREFIX = `${AWS_SCHEME}://aws-toolkit-vscode/`
```